.readthedocs.yaml

```diff
@@ -0,0 +1,32 @@
+# .readthedocs.yaml
+# Read the Docs configuration file
+# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details
+
+# Required
+version: 2
+
+# Set the OS, Python version and other tools you might need
+build:
+  os: ubuntu-22.04
+  tools:
+    python: "3.10"
+    # You can also specify other tool versions:
+    # nodejs: "19"
+    # rust: "1.64"
+    # golang: "1.19"
+
+# Build documentation in the "docs/" directory with Sphinx
+sphinx:
+  configuration: fme/docs/conf.py
+
+# Optionally build your docs in additional formats such as PDF and ePub
+# formats:
+#    - pdf
+#    - epub
+
+# Optional but recommended, declare the Python requirements required
+# to build your documentation
+# See https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html
+python:
+   install:
+   - requirements: fme/docs/requirements-pinned.txt
```

Comment:
What's the reason to not just give the `fme/docs/requirements.txt` and `fme/requirements.txt` files? We will have to remember to recompile the `requirements-pinned.txt` when those change.

Reply:
Two reasons:
- I wasn't sure that would work (whether you could specify two files), and testing changes requires waiting for sphinx to recompile on RTD which takes a little while
- RTD strongly recommends pinning the package versions (see comment from the template directly above this), which makes sense for something as public-facing as docs.

Reply:
Okay. Yeah I do see the rationale for pinning, but given the relatively infrequent updates to this repo, probably won't make much of a practical difference either way.